src/bindgen/index.md

```diff
@@ -20,6 +20,8 @@ use Rust in a C/C++ codebase in the first place!
 
 ## Getting Set Up
 
+> **Note:** Bindgen **requires** clang. Installation instructions can be found [here](https://github.com/rust-lang-nursery/rust-bindgen/blob/master/book/src/requirements.md#installing-clang-39)
```

Comment:
Although, since they're different web pages the diff isn't useful.
They have the same amount of lines apart of the warning about the mathjax url being retired.